workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/Injection/WorldCommandRequestSender.cs

```diff
@@ -60,6 +61,34 @@ public long ReserveEntityIds(uint numberOfEntityIds, uint? timeoutMillis = null,
                     return request.RequestId;
                 }
 
+                /// <summary>
+                ///     Sends a ReserveEntityIds command request.
+                /// </summary>
+                /// <param name="numberOfEntityIds">The number of entity IDs to reserve.</param>
+                /// <param name="callback">
+                ///     A callback that will be called with the command response.
+                /// </param>
+                /// <param name="timeoutMillis">
+                ///     (Optional) The command timeout in milliseconds. If not specified, will default to 5 seconds.
+                /// </param>
+                /// <returns>The request ID of the command request.</returns>
+                public long ReserveEntityIds(uint numberOfEntityIds, Action<ReserveEntityIds.ReceivedResponse> callback,
```

Comment:
It looks like if the user calls the above method with a `context = Action<WorldCommands.ReserveEntityIds.ReceivedResponse>` that will effectively do the same thing as this method.

Why not instruct the users in the above method's documentation (and other places), to pass in a `Action<WorldCommands.ReserveEntityIds.ReceivedResponse>` if they would like that callback to be called?

Reply:
I understand.